tests/test_pixel_nms.py

```diff
@@ -0,0 +1,107 @@
+# Copyright 2020 - 2021 MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+import numpy as np
+import torch
+from parameterized import parameterized
+
+from monai.utils import PixelNMS
+
+probs_map_1 = np.random.rand(100, 100).clip(0, 0.5)
```

Comment:
```suggestion
probs_map_1 = np.random.rand(100, 100)
```
Is there any reason not to keep the probability maps between 0 and 1? In the case of this NMS function, each pixel/voxel is an independent variable.